.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```

Comment:
Not sure this is needed in the project at this time, for simplicity can we leave it out?

Reply:
Without the single quotes spaces in the path variables are interpreted as parameter separator. This causes error. 
You are the owner of the GitHub repository, so you can choose. ;-)
If you leave it out, I'll keep my version. ;-)